rules-tests/DogFood/Rector/Closure/UpgradeRectorConfigRector/Fixture/paths_and_skip.php.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Rector\Tests\DogFood\Rector\Closure\UpgradeRectorConfigRector\Fixture;
+
+use Rector\Core\Configuration\Option;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
```

Comment:
Could you add $containerConfigurator->import() and/or set before to ensure method call position set is after import? Because I tried in config/set/up-to and it cause flipped set phpVersion up before set/import so need manual move down

ref

https://github.com/rectorphp/rector-src/blob/40e09ab2810dec7659f71cfb91853a023e4dff70/config/set/level/up-to-php71.php#L13-L14

Reply:
I'll try. Could you add failing test fixture?

Reply:
I will try

Reply:
I created failing test fixture PR https://github.com/rectorphp/rector-src/pull/2111 for it.